Formula/j/jj.rb

```diff
@@ -4,7 +4,7 @@ class Jj < Formula
   url "https://github.com/martinvonz/jj/archive/refs/tags/v0.17.1.tar.gz"
   sha256 "653e1190a82a39dae733487a11aa641ba62a36ae1e221c21844f822575961919"
   license "Apache-2.0"
-  revision 1
+  revision 2
```

Comment:
We can probably skip the `revision` here. Users who don't want `watchman` can do something like
```
brew reinstall jj
brew uninstall jj
brew autoremove
```

Reply:
Done and force pushed as c856eab6.